chapters/scoping.tex

```diff
@@ -166,6 +166,10 @@ \section{Instance Hierarchy Name Lookup of Inner Declarations}\doublelabel{insta
 with the same name but using the prefix \lstinline!inner! which is nearest in the
 enclosing instance hierarchy of the \lstinline!outer! element declaration.
 
+Outer component declarations may not have modifications {[}\emph{including declaration equations}{]}. 
```

Comment:
Yes. Bbinding equations include both declaration equations and hierarchical modifiers; and since we want to forbid both it should use binding equations.